src/components/main/actionsPanel/navigatorPanel/helpers/getFileNameFromPath.ts

```diff
@@ -0,0 +1,5 @@
+import { TFile } from "@_types/main";
```

Comment:
instead of having multiple files related to helpers let's have one file that exports every function in it.

What you can do to simply import is.

create a file helpers.ts inside helpers and also create and index.ts inside it

Now in index.ts do 

export * from "./helpers.ts"
export {default} from "./helpers.ts"


Now you can do named import of all the helpers function as:

import { helper1, helper2} from "./helpers"

If you have any doubt please do ask me and any suggestions if it may not be a better approach is also welcome